### DISCUSSION TITLE
GitHub "Fine-grained personal access" token

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#8449] [Q&A] [answered]
GitHub has this new [fine-grained personal access for tokens](https://github.blog/2022-10-18-introducing-fine-grained-personal-access-tokens-for-github/) which can be found under [Settings](https://github.com/settings/profile) > [Developer settings](https://github.com/settings/personal-access-tokens/new) > [Personal access tokens](https://github.com/settings/tokens?type=beta).

I looked the Weblate Docs but haven't found instructions on this.

What permissions are required for using with Weblate? Is **Contents** and **Pull requests** with "Access: Read and write" enough? (in the second case, for Weblate opening pull requests.)

**Comment by nijel** [comment, marked as answer]
Yes, those permissions should be enough.
